_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -20,7 +20,9 @@ To define a detector:
 
 1. On the **Security Analytics** home page or the **Detectors** page, choose **Create detector**.
 1. Give the detector a name and, optionally, a description. 
-1. In the **Data source** section, select one or more sources for the log data. Use an asterisk (*) to indicate a wildcard pattern. When selecting multiple data sources, their logs must be of the same type. We recommend creating separate detectors for different log types.
+1. In the **Data source** section, select one or more sources for the log data. Use an asterisk (*) to indicate a wildcard pattern. When selecting multiple data sources, their logs must be of the same type. We recommend creating separate detectors for different log types. 
+
+   Data sources support the use of [aliases]({{site.url}}{{site.baseurl}}/im-plugin/index-alias/). However, if you're configuring an alias as data source, it must be attached to a **Write** index alias, since Security Analytics only supports [data stream]({{site.url}}{{site.baseurl}}/im-plugin/data-streams/) for time-series generated data. Furthermore, when using an alias, ensure that your documents are ingesting through the alias and **not** the index(es) the alias was created for. 
```

Comment:
This part `since Security Analytics only supports [data stream]` might be a little misleading? We support both aliases and data streams independently. A data stream by definition will have a `write` index. However, aliases in opensearch are also allowed to point to read only indices which is why we want to ensure that if a detector is configured on an alias, then it needs point to at least one write index.